core/primitives/src/views.rs

```diff
@@ -1505,6 +1506,9 @@ impl From<StateChangeCause> for StateChangeCauseView {
             }
             StateChangeCause::UpdatedDelayedReceipts => Self::UpdatedDelayedReceipts,
             StateChangeCause::ValidatorAccountsUpdate => Self::ValidatorAccountsUpdate,
+            StateChangeCause::Migration { description } => {
+                Self::Migration { description: description.clone() }
```

Comment:
Seems like this'll work

```suggestion
                Self::Migration { description }
```